docs/content/content/content-structure.mdx

```diff
@@ -50,6 +50,8 @@ Use a bulleted list when there’s more than one item of a similar type. Use a n
 
 Use meaningful headings and labels so they are more accessible to all users. Headings should clearly state what the user will find on the page or paragraph that follows.
 
+It's common for screen reader users to navigate content by jumping from one heading to the next. You can nest heading levels to create subsections of content, but you should not skip heading levels as this will confuse some users.
```

Comment:
```suggestion
It’s common for screen reader users to navigate content by jumping from one heading to the next. You can nest heading levels to create subsections of content, but you should not skip heading levels as this will confuse some users.
```

Reply:
I think we need to be stronger in our language that you can't skip heading levels, and say why, something like:

> "It’s common for screen reader users to understand a page’s context and hierarchy through its heading structure, and to navigate content by jumping from one heading to the next. You can nest heading levels to create subsections of content, but you cannot not skip heading levels as this will confuse some users.

Reply:
```suggestion
It’s common for screen reader users to understand a page’s context and hierarchy through its heading structure, and to navigate content by jumping from one heading to the next. You can nest heading levels to create subsections of content, but you cannot not skip heading levels as this will confuse some users.
```

☝️ I've just changed @stowball 's text into a suggestion. This is the [same block of text used in PR 1751](https://github.com/agriculturegovau/agds-next/pull/1751/files#r1780447623)